### PR TITLE
Updated VirtualMachineImageService class 

### PR DESCRIPTION
**Pull request by djberg96**
Addresses https://github.com/ManageIQ/azure-armrest/issues/142

### CODE REVIEW

File: lib/azure/armrest/virtual_machine_image_service.rb [hunk 1]
```diff
@@ -78,21 +75,21 @@ def skus(offer, location = @location, publisher = @publisher)
       #
       # Example:
       #
-      #   vmim.versions('14.04.2', 'UbuntuServer', 'eastus', 'Canonical')
+      #   vmim.versions('15.10', 'UbuntuServer', 'eastus', 'Canonical').map(&:name)
       #
       #   # sample output
-      #   => ["14.04.201503090", "14.04.201505060", "14.04.201506100", "14.04.201507060"]
+      #   => ["15.10.201511111", "15.10.201511161", "15.10.201512030"]
```
Comment (bzwei) [inline lib/azure/armrest/virtual_machine_image_service.rb]: The sample output is a string array. The actual return type is an array of `ImageVersion`. What does `ImageVersion` look like?

**Comment by bzwei** [comment]
@djberg96 please address rubocop warnings

File: lib/azure/armrest/virtual_machine_image_service.rb (hunk 1 above)
Comment (djberg96) [inline lib/azure/armrest/virtual_machine_image_service.rb]: Note the ".map(&:name) at the end of the example code. :)

The ImageVersion object isn't particularly interesting as it just adds an id field.

**Comment by djberg96** [comment]
@bzwei Addressed the rubocop warnings.

**Pull request merged**